### ISSUE TITLE
Code examples in doc comments render incorrectly

### ISSUE THREAD

**Issue by jhenninger** [orchidhq/Orchid#350]
For example, the following KDoc comment

```kotlin
/**
 * ```
 * class ExampleApplication : Application {
 *     override val version: String = "1.0.0"
 *     override val name: String = "Example Kotlin Application"
 * 
 *     override fun start(vararg args: String) {
 *         for(i in 1..args.first().toInt()) {
 *             println("$i...")
 *             Thread.sleep(1000)
 *         }
 *     }
 * }
 * ```
 */
class ExampleApplication : Application {
```

results in: 

![image](https://user-images.githubusercontent.com/197274/72430031-93f03500-3791-11ea-9f72-f51d1cd9e9f5.png)

This happens regardless of the used theme.

**Comment by cjbrooks12** [comment]
Thanks for the report, I'll take a look and get it fixed shortly.

**Comment by cjbrooks12** [comment]
This has been fixed in 0.19.0
